.gitignore

```diff
@@ -1,4 +1,4 @@
-dist/
+/dist/
```

Comment:
Why?

Reply:
Adding the leading slash means that Git will only ignore the `dist` folder in the root of the project. Without this change, the folder `types/dist` is also ignored